src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
```

Comment:
nit: Configuration/Platform are essentially dead properties as they are set in dir.props. If we need a specific setting they should go above dir.props import.

OutputType property is also not necessary.